app/src/main/java/org/desarrolladorslp/teckersapp/ui/messages/MessageAdapter.kt

```diff
@@ -56,6 +64,7 @@ data class MessageAdapter(private val messagesHeader: ArrayList<MessageHeader>)
             Log.d("RecyclerViewMessage", "CLICK!")
         }
 
+        @RequiresApi(Build.VERSION_CODES.O)
```

Comment:
I don't think this is ideal, less than 20% of Mexican Android devices have Android O or higher